### DISCUSSION TITLE
Show how many seconds left on 60 second rule

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2122] [Ideas]
If it's possible, it should happen 😃

**Comment by retronbv** [comment]
What 60 second rule?

**Comment by cobaltt7** [comment]
You need to wait 60 seconds (120 for new scratchers) between posts on the forums.

**Reply by retronbv** [reply]
Ah, Im not really a forums user

**Reply by ItsEnderium** [reply]
Many Scratch Addons users are Forum Helpers.

**Comment by GrahamSH-LLK** [comment]
This isn't as easy as it seems. By the time we get back a response saying that the 60 seconds rule has been triggered, a random amount of time has passed.
